docs/src/index.md

```diff
@@ -45,3 +45,22 @@ To load the package, use
 ```julia
 using Ferrite
 ```
+
+## Debugging Information
```

Comment:
Ideally this would be on a separate `Developer documentation` page in the future, looks a bit out of place here, but good for now.

Reply:
I also started with a developer information page first and agree that for now that is too empty. Will try to put something together after getting the two big open PRs on track. :)